src/libs/backends/winit.rs

```diff
@@ -144,10 +137,10 @@ pub fn winit_dispatch(
 
 	*full_redraw = full_redraw.saturating_sub(1);
 
-	let size = winitdata.backend.window_size().physical_size;
+	let size = state.backend.window_size().physical_size;
 	let damage = Rectangle::from_loc_and_size((0, 0), size);
 
-	winitdata.backend.bind().unwrap();
+	state.backend.bind().unwrap();
```

Comment:
Is it not possible to use anyhow in this function?

Reply:
most probably yes. we need to implement anyhow in a lot of place so i'll work on it and create a new PR.